dist/gcc-compatible/Hacl_Hash_Blake2b.h

```diff
@@ -0,0 +1,142 @@
+/* MIT License
+ *
+ * Copyright (c) 2016-2022 INRIA, CMU and Microsoft Corporation
+ * Copyright (c) 2022-2023 HACL* Contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+
+#ifndef __Hacl_Hash_Blake2b_H
+#define __Hacl_Hash_Blake2b_H
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+#include <string.h>
+#include "krml/internal/types.h"
+#include "krml/lowstar_endianness.h"
+#include "krml/internal/target.h"
+
+#include "Lib_Memzero0.h"
+#include "Hacl_Krmllib.h"
+
+void Hacl_Hash_Blake2b_init(uint64_t *hash, uint32_t kk, uint32_t nn);
+
+void
+Hacl_Hash_Blake2b_update_key(
```

Comment:
These functions need documentation.

Reply:
They are here, so they are in the public API 😉

Reply:
`Hacl_Hash_Blake2b_init` is used in `Hacl_Hash_Blake2b_hash_with_key` and the `init` function of `EverCrypt_Hash` as well as in `Hacl_HMAC_compute_blake2b_32`.

`Hacl_Hash_Blake2b_update_key` is only used in `Hacl_Hash_Blake2b.c`.